cli/cli.go

```diff
@@ -1,5 +1,7 @@
 package cli
 
+import "os"
+
 var _stopProfiling func()
 
 func stopProfiling() {
```

Comment:
Maybe leave a comment here that sync.Once is used inside `_stopProfiling` or even use it on this func itself.